src/components/Menu/Menu.jsx

```diff
@@ -57,6 +65,15 @@ class Menu extends Component {
           </div>
           <div className={openNav ? "navbar-menu is-active" : "navbar-menu"}>
             <div className="navbar-start">{items}</div>
+            <div className="navbar-end">
+              <div className="navbar-item">
+                <div className="buttons">
+                  <a className="button is-primary">
+                    <strong>Вход</strong>
```

Comment:
```suggestion
                    <strong onClick={this.props.toggleModal}>Вход</strong>
```
для работы кнопки "вход":)